schema.sql

```diff
@@ -0,0 +1,61 @@
+
+-- Table for movies
+CREATE TABLE movies (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  title VARCHAR(255),
+  genre VARCHAR(255),
+  author VARCHAR(255),
+  source VARCHAR(255),
+  label VARCHAR(255),
+  publish_date DATE,
+  archived BOOLEAN
+);
+
+-- Table for sources
+CREATE TABLE sources (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  name VARCHAR(255) 
+)
+-- Create the books table by using the following schema
+
+CREATE TABLE books (
+  id INT PRIMARY KEY,
+  genre VARCHAR(255),
+  author VARCHAR(255),
+  source VARCHAR(255),
+  label_id INT,
+  publish_date DATE,
+  archived BOOLEAN,
+  FOREIGN KEY (label_id) REFERENCES labels(id)
+);
+
+-- Create the labels table by using the following schema
+CREATE TABLE labels (
+  id INT PRIMARY KEY,
+  title VARCHAR(255),
+  color VARCHAR(255)
+
+);
+-- Table for games
+CREATE TABLE games (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  title VARCHAR(255),
+  genre VARCHAR(255),
+  author VARCHAR(255),
+  source VARCHAR(255),
+  label VARCHAR(255),
+  publish_date DATE,
+  archived BOOLEAN,
+  multiplayer BOOLEAN,
+  last_played_at DATE
+);
```

Comment:
- [ ]  Please instead of VARCHAR(255), the `author`, `label`, and `genre` columns are supposed to be foreign keys which references the id on their respective tables. This is how you ensure the belongs-to relationship between the games and the aforementioned classes.